tests/test_StaticSVD.C

```diff
@@ -0,0 +1,88 @@
+#include "../StaticSVDSampler.h"
+#include <algorithm>
+#include <cmath>
+#include <cstdio>
+#include <cstring> // for memcpy
+#include <random>
+
+#include "mpi.h"
+
+int main(int argc, char* argv[])
+{
+    MPI_Init(&argc, &argv);
```

Comment:
We should migrate this test to the unit testing suite for consistency; those tests use [googletest](https://github.com/google/googletest). There aren’t any parallel unit tests at the moment, but writing an example of a parallel unit test shouldn’t be too hard, and it would give you an idea of what we’d be aiming for.

Reply:
I am happy to do this but if it's ok with you, let's go ahead with this PR before setting it up as a unit test in the framework.

Reply:
That's fine -- we can leave that to another PR, as long as it gets done by, say, end of August 2019.